src/SFA.DAS.ApprenticeCommitments/Configuration/ApplicationSettings.cs

```diff
@@ -1,9 +1,12 @@
 ﻿#nullable disable
 
+using System;
+
 namespace SFA.DAS.ApprenticeCommitments.Configuration
 {
     public class ApplicationSettings
     {
         public string DbConnectionString { get; set; }
+        public TimeSpan TimeToWaitBeforeChangeOfApprenticeshipEmail { get; set; } = TimeSpan.FromHours(24);
```

Comment:
Just curious, what's a TimeSpan configuration value going to look like? Will it de-serialise correctly. I've not used one before

Reply:
For hours it's simply "24:00:00", which seems intuitive to me.

You can use larger time spans by using the formats in the [documentation](https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-timespan-format-strings).

Reply:
Don't think we now need this, but it's up to you if you want to remove it now or later